allennlp/data/token_indexers/elmo_indexer.py

```diff
@@ -0,0 +1,113 @@
+from typing import Dict, List
+
+from overrides import overrides
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.params import Params
+from allennlp.common.util import pad_sequence_to_length
+from allennlp.data.tokenizers.token import Token
+from allennlp.data.token_indexers.token_indexer import TokenIndexer
+from allennlp.data.vocabulary import Vocabulary
+
+
+def _make_bos_eos(char, pad_char, bow_char, eow_char, max_word_length):
```

Comment:
Given you've created a class to do this processing below, seems like you could add this in there.

Reply:
I'm calling it to create class attributes, couldn't figure an easy way to add it as a class method and also call it there.